.github/scripts/sanatizer-git-commit-messages

```diff
@@ -22,7 +22,7 @@ echo "$commits" > commit_messages.txt
 
 # Check commit messages format
 while IFS= read -r line; do
-  if ! [[ $line =~ ^[a-zA-Z0-9\_\-]+:\ .+$ ]]; then
+  if ! [[ $line =~ ^[a-zA-Z0-9,_\-]+:\ .+$ ]]; then
```

Comment:
If $line can contain spaces, as it seems to be the case considering the regex, then $line should be quoted.